.github/workflows/push.yml

```diff
@@ -19,22 +19,19 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: ['2.5.1', '2.7.2', '2.6.5']
+        ruby: ['3.2.2']
```

Comment:
Why `3.2`? At least ruby `3.0.x` is still in `security maintenance phase` https://www.ruby-lang.org/en/downloads/ so id say its better to have the active ruby versions supported!?